polaris.shopify.com/src/components/PatternsExample/PatternsExample.tsx

```diff
@@ -282,6 +283,7 @@ const PatternsExample = ({
           defaultHeight={defaultHeight}
           minHeight={minHeight}
           src={previewUrl}
+          sandbox="allow-scripts"
```

Comment:
This change causes the auto-height-adjustment to break (because there's no longer a `window.parent` or `window.origin` for the iframe to send messages to.

This small breakage is a low priority thing we can fix in a follow-up PR.